spec/decorators/hash_diff_decorator_spec.rb

```diff
@@ -24,5 +24,10 @@
       output = HashDiffDecorator.html_for(['~', 'foo', '', 'bar'])
       expect(output).to eq("<code>foo</code> was changed from <code>[empty]</code> to <code>&quot;bar&quot;</code>")
     end
+
+    it "renders numeric events" do
+      output = HashDiffDecorator.html_for(['~', 'a_number', 456, 123])
+      expect(output).to eq("<code>a_number</code> was changed from <code>456.00</code> to <code>123.00</code>")
```

Comment:
is this test for the case above? seems to cover a numeric rather than empty val?